prody/proteins/blastpdb.py

```diff
@@ -1,50 +1,73 @@
 # -*- coding: utf-8 -*-
-"""This module defines functions for blast searching Protein Data Bank."""
+"""This module defines functions for blast searching the Protein Data Bank."""
 
 import os.path
-import os
 
-from prody import LOGGER
+from prody import LOGGER, PY3K
 from prody.utilities import dictElement, openURL, which
+from prody.sequence.msafile import parseMSA
 
-__all__ = ['PDBBlastRecord', 'blastPDB', 'showSequenceTree']
+import platform, os, re, sys, time, urllib
+
+if PY3K:
+    import urllib.parse as urllib
+    import urllib.request as urllib2
+else:
+    import urllib
+    import urllib2
 
+from prody.sequence import Sequence
+from prody.atomic import Atomic
+from prody.proteins.pdbfile import parsePDB
+
+__all__ = ['PDBBlastRecord', 'blastPDB', 'showSequenceTree']
 
-def blastPDB(sequence, filename=None, **kwargs):
+def blastPDB(sequence='runexample', filename=None, **kwargs):
```

Comment:
Let's leave `sequence` as a required/positional argument. It can make it clearer that `sequence` is the only and crucial argument of this function without needing to read the source code or the document. It may also help the users understand the function of this function is to blast a sequence against the PDB. Leaving some positional arguments is generally (not always) a [good practice](https://stackoverflow.com/questions/7041752/any-reason-not-to-always-use-keyword-arguments).